expressEngine/model/lib/index.js

```diff
@@ -9,7 +9,7 @@ class ModelLib {
 
   };
   validatePassword = (password) => {
-    var re = /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[#$@!%&*?])[A-Za-z\d#$@!%&*?]{8,50}$/;
+    var re = /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[#$@!%&*?])[A-Za-z\d!"#$%&'()*+,-./:;<=>?@[\]^_`{|}~]{8,50}$/;
```

Comment:
Please check the regex again, it is not accepting many special characters

Reply:
please mention those special characters